.changeset/stupid-humans-fry.md

```diff
@@ -0,0 +1,5 @@
+---
+'@firebase/messaging': patch
+---
+
+Tag product id for big query metric logging
```

Comment:
Can you add internal, e.g. "Tag product id for internal bigquery metric logging"?